sass/_main.scss

```diff
@@ -143,7 +146,8 @@ footer nav {
 
 body.essay {
     h1 {
-        @extend .f1, .fw3;
+        font-size: 1rem;
```

Comment:
This was my mistake — maybe the h1 inside `templates/essay/post.html` should just have the `fw6` class? Then its h1 will just inherit the flattening from the global h1 styling and we won't need to declare it here.

Reply:
Yeah, thats prob a good idea, will do